.gitignore

```diff
@@ -22,3 +22,13 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 chat_app-*.tar
 
+/_build
```

Comment:
Can you remove the duplicates from the gitignore?

Reply:
Woops. I thought `gitignore` was smarter than that 😬

Reply:
Haha, yeah would be nice if it was!